pcsx2/gui/Panels/SpeedhacksPanel.cpp

```diff
@@ -281,7 +281,8 @@ void Panels::SpeedHacksPanel::EnableStuff( AppConfig* configToUse )
 	m_check_waitloop->Enable(HacksEnabledAndNoPreset);
 	m_check_fastCDVD->Enable(HacksEnabledAndNoPreset);
 
-	m_check_vuThread->Enable(hacksEnabled); // MTVU is unaffected by presets
+	// Grayout MTVU on safest preset
+	m_check_vuThread->Enable(hacksEnabled && ((configToUse->PresetIndex != 0 && hasPreset) || !hasPreset));
```

Comment:
I think this could be simplified to:
`m_check_vuThread->Enable(hacksEnabled && (!hasPreset || configToUse->PresetIndex != 0));`